formulaire/src/main/resources/public/ts/directives/question/question-item/question-item.directive.ts

```diff
@@ -128,6 +129,18 @@ class Controller implements IViewModel {
         let nextElements: FormElement[] = this.formElements.all.filter((e: FormElement) => e.position === this.getPosition() + 1);
         return nextElements.length > 0 ? nextElements[0].id : null;
     }
+
+    imageHasLabel = () : boolean => {
+        const question = this.question;
+        const choices = question.choices.all;
+
+        for (const choice of choices) {
+            if (choice.image !== null && choice.value === "") {
+                return true;
+            }
+        }
+        return false;
+    };
```

Comment:
```suggestion
    imageHasLabel = () : boolean => {
        const choices = this.question.choices.all;
        return choices.find((choice: Typage) => choice.image !== null && choice.value === "").length > 0;
    };
```

Reply:
Alice m'avait déjà suggérer de simplifier comme ça : 
const choices: QuestionChoice[] = this.question.choices.all.filter(choice => choice.image !== null
            && (choice.value === "" || choice.value === null || choice.value === undefined))
        return choices.length > 0;

Reply:
Je pense quand dans mon besoin ta deuxième proposition correspondra mieux car je n'ai pas besoin d'accéder aux éléments mais seulement de savoir si un élément correspondant existe